### PR TITLE
Fix test lint. Remove gh actions in favor of prow

### PR DESCRIPTION
**Pull request by pacevedom**
Fix linting in `test` dir. Remove github actions in favor of prow.

Relates to:
https://github.com/openshift/release/pull/24069
https://github.com/openshift/release/pull/24099
https://github.com/openshift/release/pull/24101

### CODE REVIEW

File: test/e2e/basic/specialresource_test.go [hunk 1]
```diff
@@ -14,8 +14,6 @@ import (
 
 const (
 	NamespaceSRO = "openshift-special-resource-operator"
-	pollInterval = 1 * time.Second
-	waitDuration = 10 * time.Minute
```
Comment (yevgeny-shnaidman) [inline test/e2e/basic/specialresource_test.go:14]: why did you remove it from here, if the values for simple-kmod and ping-pong are the same

Comment (pacevedom) [reply test/e2e/basic/specialresource_test.go:14]: `lint` and `verify` targets complain about it. Besides, this adds flexibility to change them to different values if needed.

Comment (qbarrand) [reply test/e2e/basic/specialresource_test.go:14]: @pacevedom please have a look at https://github.com/openshift-psap/special-resource-operator/pull/160 where we make all e2e tests `_test.go` files.

Comment (qbarrand) [reply test/e2e/basic/specialresource_test.go:14]: In fact, should we merge #160 with this PR?

**Comment by ybettan** [comment]
I am wondering why the CI isn't running on this PR, maybe it will always work once the PR is merged?

**Comment by qbarrand** [comment]
@ybettan No one commented `/ok-to-test`
/ok-to-test

**Comment by ybettan** [comment]
I meant that this PR is supposed to make all other miss-configured CI jobs to work again, but they aren't triggered in this PR @qbarrand , @pacevedom .

I have tried it myself, before this PR was created and I got the same issue so I thought that this isn't the correct fix.

Maybe I am missing something here.
https://github.com/openshift-psap/special-resource-operator/pull/161
